hoprd/db/api/src/db.rs

```diff
@@ -0,0 +1,82 @@
+use hoprd_migration::{MigratorMetadata, MigratorTrait};
+use sea_orm::SqlxSqliteConnector;
+use sqlx::pool::PoolOptions;
+use sqlx::sqlite::{SqliteAutoVacuum, SqliteConnectOptions, SqliteJournalMode, SqliteSynchronous};
+use sqlx::{ConnectOptions, SqlitePool};
+use std::path::Path;
+use std::time::Duration;
+use tracing::log::LevelFilter;
+
+use crate::HoprdDbAllOperations;
+
+#[derive(Debug, Clone)]
+pub struct HoprdDb {
+    pub(crate) metadata: sea_orm::DatabaseConnection,
+}
+
+pub const SQL_DB_METADATA_FILE_NAME: &str = "hopr_metadata.db";
```

Comment:
I'd just go with `hoprd`, but if you want to split it, it's okay. Just make sure there's only ever a single write instance that can lock you out.